operations/helm/tests/build.sh

```diff
@@ -17,7 +17,11 @@ TESTS=$(ls -1 ${CHART_PATH}/ci/*values.yaml)
 for FILEPATH in $TESTS; do
   # Extract the filename (without extension).
   TEST_NAME=$(basename -s '.yaml' "$FILEPATH")
+  OUTPUT_DIR="operations/helm/tests/${TEST_NAME}-generated"
 
   echo "Templating $TEST_NAME"
-  helm template "${TEST_NAME}" ${CHART_PATH} -f "${FILEPATH}" --output-dir "operations/helm/tests/${TEST_NAME}-generated" --namespace citestns
+  helm template "${TEST_NAME}" ${CHART_PATH} -f "${FILEPATH}" --output-dir "${OUTPUT_DIR}" --namespace citestns
+
+  echo "Removing mutable config checksum and helm chart version for clarity in ${OUTPUT_DIR}"
+  find "${OUTPUT_DIR}" -type f -print0 | xargs -0 sed -E -i -- "/^(([ ]+checksum\/config: [0-9a-f]{64})|([ ]+(helm.sh\/)?chart: mimir-distributed-([0-9]|[1-9][0-9]+)[.]([0-9]|[1-9][0-9]+)[.]([0-9]|[1-9][0-9]+)(-weekly.([0-9]|[1-9][0-9]+))?))$/d"
```

Comment:
nit: I would simplify this regex to just remove lines with `helm.sh/chart` and `checksum/config` rather than putting the semver regex in here.

Reply:
ok, may have gone a bit overboard with that one, WDYT of the current version? it does affect Minio manifests as well

Reply:
Looks good. I think you could run the sed on `"${OUTPUT_DIR}/templates" to avoid minio, right? Either way is fine with me